### DISCUSSION TITLE
Managing Disk Space for T-Pot in High-Traffic Environments

### DISCUSSION THREAD

**Discussion by kanowah** [telekom-security/tpotce#1632] [General]
I recently installed T-Pot in my organization, but we experience a high volume of traffic, causing the T-Pot hard disk to fill up within 24 hours. Once the hard disk is full, the system crashes. Is there a solution to purge old data or a cron job that can help clean up the system regularly, such as every month or year?

**Comment by t3chn0m4g3** [comment]
Adjust [logrotate.conf](https://github.com/telekom-security/tpotce/blob/master/docker/tpotinit/dist/etc/logrotate/logrotate.conf) to your own needs, add a volume to the `docker-compose.yml` `tpotinit` service and restart T-Pot. Make sure to use the correct paths and filenames inside and outside the container.
Outside the logs on the host follow the [ReadMe](https://github.com/telekom-security/tpotce?tab=readme-ov-file#log-persistence).
